website/docs/r/policy_ip_address_allocation.html.markdown

```diff
@@ -113,5 +113,10 @@ An existing IP Allocation can be [imported][docs-import] into this resource, via
 ```
 terraform import nsxt_policy_ip_address_allocation.test POOL-ID/ID
 ```
-
 The above command imports IpAddressAllocation named `test` with the NSX IpAddressAllocation ID `ID` in IP Pool `POOL-ID`.
+
+```
+terraform import nsxt_policy_ip_address_allocation.test POLICY_PATH
+```
+The above command imports IpAddressAllocation named `test` with the NSX policy path `POLICY_PATH`.
+Note: for multitenancy projects only the later form is usable.
```

Comment:
Seems like there's only one form in this import

Reply:
Why? Non-multitenancy resources could use pool-id/allocation-id as before

Reply:
Sorry my bad, you're right